src/ConcordiumNetSdk.TestConsoleApp/Program.cs

```diff
@@ -1,10 +1,31 @@
-﻿using ConcordiumNetSdk;
+﻿using Newtonsoft.Json;
+using Concordium.V2;
+using ConcordiumNetSdk;
+using AccountAddress = ConcordiumNetSdk.Types.AccountAddress;
 
-Connection connection = new Connection
+// Create the client.
+var url = "https://service.internal.stagenet.concordium.com";
+UInt16 port = 20000;
+Client concordiumNodeClient = new Client(url, port);
+
+var address = "3EqkGQ7NvakjpbUeocRiGzao62ZEEvB6A5rjGsezL8bQBKQGU8";
+var mySender = AccountAddress.From(address).ToProto();
+var best = new BlockHashInput() { Best = new Empty() };
+
+var myAccountInfoRequest = new AccountInfoRequest()
 {
-    Address = "http://34.71.98.161:10001",
-    AuthenticationToken = "rpcadmin"
+    BlockHash = best,
+    AccountIdentifier = new AccountIdentifierInput() { Address = mySender }
 };
-ConcordiumNodeClient concordiumNodeClient = new ConcordiumNodeClient(connection);
 
+var moduleListRes = concordiumNodeClient.GetModuleList(best);
```

Comment:
This demonstrates some basic usage. I just left it for you to look at in case you have some feedback.

Reply:
This is for youuuur eyes only.

Reply:
Yes! :-)